pontoon/base/placeables.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+
+class PythonFormatNamedString:
+    parser = r"(%\([[\w\d!.,[\]%:$<>+\-= ]*\)[+|-|0\d+|#]?[.\d+]?[s|d|e|f|g|o|x|c|%])"
+
+
+class PythonFormatString:
+    parser = r"(\{{?[\w\d!.,[\]%:$<>+-= ]*\}?})"
+
+
+class PythonFormattingVariable:
+    parser = r"(%(%|(\([^)]+\)){0,1}[-+0#]{0,1}(\d+|\*){0,1}(\.(\d+|\*)){0,1}[hlL]{0,1}[diouxXeEfFgGcrs]{1}))"
+    match_index = 0
+
+
+class FluentTerm:
+    parser = r"({ ?-[^}]* ?})"
+
+
+class FluentParametrizedTerm:
```

Comment:
```suggestion
class FluentParameterizedTerm:
```

Reply:
I've never seen parametrized before, but apparently it's a thing.

For sure that's not the terminology we used in Fluent
https://projectfluent.org/fluent/guide/terms.html

Reply:
Let's make it easy for ourselves and remove this; `FluentTerm` is already capturing practically all valid parameterized terms, and the ones that are valid that it isn't capturing aren't captured by this either.